Character_Builder/models.py

```diff
@@ -35,14 +35,53 @@
 # As this is in the character builder folder, this will focus on
 # the character information 
 
+# Automatically creates CharacterIDs
+def newCharacterID():
```

Comment:
As Ben said, it may be best to either use the hidden built-in .id field, or to make your own custom id field with:

models.AutoField(primary_key=True)